### ISSUE TITLE
False negative on itextpdf 5.5.11

### ISSUE THREAD

**Issue by lrst** [dependency-check/DependencyCheck#1376]
Hi, 

I believe there is false negative on itextpdf 5.5.11.

```xml
<dependency>
   <groupId>com.itextpdf</groupId>
   <artifactId>itextpdf</artifactId>
   <version>5.5.11</version>
</dependency>
```

No alert from dependency-check despite: https://nvd.nist.gov/vuln/detail/CVE-2017-9096

The report shows:
```
cpe: cpe:/a:itextpdf:itext:5.5.11  Confidence:Low  
maven: com.itextpdf:itextpdf:5.5.11  Confidence:Highest
```

Regards,

**Comment by jeremylong** [comment]
I believe this is an issue with the information in the NIST NVD data feeds - specifically the XML data feeds that are currently being used.  The JSON feeds are better - but the change is significant on my end. Luckily - the move from the XML to JSON data feeds is next on my list.

**Comment by lrst** [comment]
After digging a bit, it seems that JSON feeds are not better in this case. Both feeds totally miss the vulnerable versions before 5.1.12: 
```
      "affects" : {
        "vendor" : {
          "vendor_data" : [ {
            "vendor_name" : "itextpdf",
            "product" : {
              "product_data" : [ {
                "product_name" : "itext",
                "version" : {
                  "version_data" : [ {
                    "version_value" : "7.0.0"
                  }, {
                    "version_value" : "7.0.1"
                  }, {
                    "version_value" : "7.0.2"
                  } ]
                }
              } ]
            }
          } ]
        }
      },
```
:(

**Comment by jeremylong** [comment]
This has been resolved with 5.0.0.
